odl/test/solvers/smooth/smooth_test.py

```diff
@@ -34,6 +32,17 @@ def functional(request):
         scaling = odl.MultiplyOperator(space.element([1, 2, 3]),
                                        domain=space)
         return odl.solvers.L2NormSquared(space) * scaling
+    elif name == 'quadratic_form':
+        space = odl.rn(3)
+        # Symmetric and diagonally dominante matrix
+        matrix = odl.MatrixOperator([[7.0, 1, 2], [1, 5, -3], [2, -3, 8]])
```

Comment:
At the cost of vertical space, matrices look nicer if the initializing nested lists are shown one row at a time.